Sources/Sharing/Internal/PassthroughRelay.swift

```diff
@@ -2,50 +2,59 @@
   import Combine
   import Foundation
 
-  final class PassthroughRelay<Output>: Publisher {
+  final class PassthroughRelay<Output>: Subject {
```

Comment:
Needed to upgrade this to a full `Subject` to pipe a new publisher into the relay.

Not 100% sure I got all of this right, so please give it a good read.